sdk/storage/azure-storage-file-share/out/azure/storage/fileshare/_models.pyi

```diff
@@ -35,6 +35,14 @@ class AccessPolicy(GenAccessPolicy):
     permission: Any = ...
     def __init__(self, permission: Optional[Any] = ..., expiry: Optional[Any] = ..., start: Optional[Any] = ...) -> None: ...
 
+class CopyFileSmbInfo(GenCopyFileSmbInfo):
```

Comment:
We have other changes that will be made to the properties of this object (see email thread) however this object and it's parameter could also be renamed to align across languages